lib/checks/aria/aria-allowed-attr-evaluate.js

```diff
@@ -82,7 +82,7 @@ function ariaAllowedAttrEvaluate(node, options, virtualNode) {
 
   if (invalid.length) {
     this.data(invalid);
-    return false;
+    return isHtmlElement(virtualNode) ? false : undefined;
```

Comment:
I think this exception should only be used:
1. If the element is not an HTML element
2. If the element is not focusable
3. If the element has no role